packages/sitecore-jss-dev-tools/src/disconnected-server/create-default-disconnected-server.ts

```diff
@@ -94,6 +94,21 @@ export function createDefaultDisconnectedServer(options: DisconnectedServerOptio
     app = Express();
   }
 
+  // backwards compatibility with fix for #80
+  // for GA the appRoot was expected to be $app/scripts
+  // which didn't make sense. This allows both sane app roots
+  // and GA-style app roots to keep working.
+  if(options.appRoot.endsWith('scripts')) {
+    options.appRoot = join(options.appRoot, '..');
+  }
+
+  // further backwards compatibility for #80
+  // allows apps with GA watch path of '../data' (relative to /scripts)
+  // to keep working even with appRoot now relative to the actual app root
+  // We do this by stripping '../' from path leads, making the path './data' instead - theoretically, the chance of
+  // wanting to actually escape from the app root entirely otherwise is awfully low.
+  options.watchPaths = options.watchPaths.map(path => path.startsWith('../') ? path.substring(1) : path);
```

Comment:
Are `watchPaths` always relative to `appRoot`? If so, might be worth adding a note to the `watchPaths` option comment (line 18) that they're expected to be relative.

Reply:
They can be but don't have to be (i.e. `c:\foo` would also be valid) - but if they are, they are app-root-relative, yes.

Reply:
Added the comment. Good suggestion.